SPEC.md

```diff
@@ -85,7 +91,7 @@ Success is indicated by a return code of zero and the following JSON printed to
 
 ```
 {
-  "cniVersion": "0.2.0",
+  "cniVersion": "0.3.0",
```

Comment:
Are we bumping the CNI spec version to 0.3?  Do we need to do that, given we haven't had a CNI release yet with version 0.2?